src/rsz/src/RemoveBuffer.cc

```diff
@@ -0,0 +1,169 @@
+/////////////////////////////////////////////////////////////////////////////
+//
+// Copyright (c) 2019, The Regents of the University of California
```

Comment:
Copyright should be updated to 2024 and Precision Innovations Inc.

Reply:
Fixed